app/src/main/java/org/ole/planet/myplanet/ui/dashboard/DashboardActivity.kt

```diff
@@ -602,6 +602,7 @@ class DashboardActivity : DashboardElementActivity(), OnHomeItemClickListener, N
             ResourcesCompat.getDrawable(resources, R.drawable.team, null)?.let { menuImageList.add(it) }
             ResourcesCompat.getDrawable(resources, R.drawable.business, null)?.let { menuImageList.add(it) }
             ResourcesCompat.getDrawable(resources, R.drawable.survey, null)?.let { menuImageList.add(it) }
+            ResourcesCompat.getDrawable(resources, R.drawable.survey, null)?.let { menuImageList.add(it) }
```

Comment:
Hi @Kuljeet1998 , line 604 and 605 are both adding the survey drawable. Are we adding the icon twice here?

Reply:
Yes we add it twice so that both "community" and "survey" have 2 separate icons and when they are clicked only that particular changes instead of both.